index.js

```diff
@@ -52,8 +71,11 @@ module.exports = async function (appConfig) {
   if (config.logger == null)
     config.logger = true
   else if (config.logger !== true && appConfig.elastic)
-    // If elastic is configured, use pine with pine-elasticsearch
-    config.logger = setupLogging(appConfig.elastic, config.logger, appConfig.serviceName)
+    // If elastic is configured, use pino with pino-elasticsearch
+    config.logger = setupElasticLogging(appConfig.elastic, config.logger, appConfig.serviceName)
+  else if (config.logger !== true && appConfig.logtail)
```

Comment:
Double check for config.logger !== true, nest instead

Reply:
nesting increases code complexity by 2, wontfix

Reply:
will create an separate function setupLogging